config/game_options.txt

```diff
@@ -180,44 +180,44 @@ MIDROUND_ANTAG WIZARD
 ## Notes: For maximum number of players a value of -1 means no maximum. Setting minimums to
 ## VERY low numbers (< 5) can lead to errors if the roundtypes were not designed for that.
 
-#MIN_POP TRAITOR 0
-#MAX_POP TRAITOR -1
+MIN_POP TRAITOR 0
+MAX_POP TRAITOR -1
 
-#MIN_POP TRAITORBRO 0
-#MAX_POP TRAITORBRO -1
+MIN_POP TRAITORBRO 10
+MAX_POP TRAITORBRO -1
 
-#MIN_POP TRAITORCHAN 15
-#MAX_POP TRAITORCHAN -1
+MIN_POP TRAITORCHAN 15
+MAX_POP TRAITORCHAN -1
 
-#MIN_POP DOUBLE_AGENTS 25
-#MAX_POP DOUBLE_AGENTS -1
+MIN_POP DOUBLE_AGENTS 25
+MAX_POP DOUBLE_AGENTS -1
 
-#MIN_POP NUCLEAR 0
-#MAX_POP NUCLEAR -1
+MIN_POP NUCLEAR 0
+MAX_POP NUCLEAR -1
 
-#MIN_POP REVOLUTION 20
-#MAX_POP REVOLUTION -1
+MIN_POP REVOLUTION 20
+MAX_POP REVOLUTION -1
 
-#MIN_POP CULT 24
-#MAX_POP CULT -1
+MIN_POP CULT 24
+MAX_POP CULT -1
 
-#MIN_POP CHANGELING 15
-#MAX_POP CHANGELING -1
+MIN_POP CHANGELING 15
+MAX_POP CHANGELING -1
 
-#MIN_POP WIZARD 20
-#MAX_POP WIZARD -1
+MIN_POP WIZARD 20
+MAX_POP WIZARD -1
 
-#MIN_POP MONKEY 20
-#MAX_POP MONKEY -1
+MIN_POP MONKEY 20
+MAX_POP MONKEY -1
 
-#MIN_POP METEOR 0
-#MAX_POP METEOR -1
+MIN_POP METEOR 0
+MAX_POP METEOR -1
 
-#MIN_POP DEVIL 0
-#MAX_POP DEVIL -1
+MIN_POP DEVIL 20
+MAX_POP DEVIL -1
 
-#MIN_POP DEVIL_AGENTS 25
-#MAX_POP DEVIL_AGENTS -1
+MIN_POP DEVIL_AGENTS 25
```

Comment:
Devil is disabled

Reply:
This doesn't change that, so the override being set is fine.